packages/office-addin-dev-certs/package.json

```diff
@@ -17,7 +17,14 @@
     "office-addin-dev-certs": "./cli.js"
   },
   "keywords": [
```

Comment:
Should "Add-in" be added in? :-)

Some of these like "development", "server", and "secure" are very generic and seem of little value for search purposes.  

"self-signed" seems misleading, since they aren't self-signed.

"localhost" might also be good to add.

Reply:
If people search like "development server certificate" this package should be suggested right. Generally speaking, I've seen people adding so many keywords for better discoverability. (Eg https://www.npmjs.com/package/https-pem). I'll add "add-in" and "localhost" and remove "self-signed"